### DISCUSSION TITLE
v1.7.0-beta.6

### DISCUSSION THREAD

**Discussion by EvanMulawski** [EvanMulawski/FanControl.CorsairLink#212] [Announcements]
- iCUE LINK: Fixes an issue preventing device detection.

<hr /><em>This discussion was created from the release <a href='https://github.com/EvanMulawski/FanControl.CorsairLink/releases/tag/v1.7.0-beta.6'>v1.7.0-beta.6</a>.</em>

**Comment by XeTrinity** [comment]
Snice installing beta 6 ive been getting "missing control sensor" on all my fans/pump and nothing is showing in fancontrol .net8
[CorsairLink.log](https://github.com/user-attachments/files/18204023/CorsairLink.log)

**Reply by EvanMulawski** [reply]
Hi @ScoPeZs, do you know what firmware version the hub is running?

**Comment by XeTrinity** [comment]
I'ts running 2.9.488.

**Reply by EvanMulawski** [reply]
This will be fixed in v1.7.0-beta.7 - should be available shortly.

**Reply by XeTrinity** [reply]
awesome beta 7 seems to have fixed the issue thank you
